### PR TITLE
test(dashmate): successful migration test

### PR DESCRIPTION
**Pull request by pshenmic**
## Issue being fixed or feature implemented

Add a test that will run the migration config code and check if final config is valid, executable and contains right image versions

## What was done?

* Added a unit test for successful migration
* Added a e2e test for --platform flag on start / stop evo nodes

## How Has This Been Tested?

Locally

## Breaking Changes

No

## Checklist:

- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have added or updated relevant unit/integration/functional/e2e tests
- [x] I have added "!" to the title and described breaking changes in the corresponding section if my code contains any
- [x] I have made corresponding changes to the documentation if needed

**For repository code-owners and collaborators only**
- [x] I have assigned this pull request to a milestone

### CODE REVIEW

**Comment by pshenmic** [comment]
https://github.com/dashpay/platform/issues/1275

File: packages/dashmate/test/unit/commands/config/configFile/migrateConfigFileFactory.js [hunk 1]
```diff
@@ -0,0 +1,79 @@
+const _ = require('lodash');
```
Comment (shumkov) [inline packages/dashmate/test/unit/commands/config/configFile/migrateConfigFileFactory.js:1]: .spec.js

File: packages/dashmate/configs/defaults/getTestnetConfigFactory.js [hunk 2]
```diff
@@ -26,7 +26,7 @@ function getTestnetConfigFactory(homeDir, getBaseConfig) {
       },
       core: {
         docker: {
-          image: 'dashpay/dashd:20.0.0-beta.2',
+          image: 'dashpay/dashd:20.0.0-beta.4',
```
Comment (shumkov) [inline packages/dashmate/configs/defaults/getTestnetConfigFactory.js:29]: Please update version a separate PR

**Review by shumkov — CHANGES_REQUESTED** [review]

**Review by shumkov — APPROVED** [review]
👍

**Pull request merged**